packages/esm-patient-registration-app/src/patient-registration/field/id/id-field.component.tsx

```diff
@@ -113,7 +113,13 @@ export const Identifiers: React.FC = () => {
 
   return (
     <div className={styles.halfWidthInDesktopView}>
-      <UserHasAccess privilege="coreapps.systemAdministration">
+      <UserHasAccess
```

Comment:
Hi @jona42-ui !
Actually the User has access needs to wrap the button below only, and not the whole header. The previous wrap is not correct.

If there is not sufficient privelege, it'll hide the `Identifiers` heading as well from the form, but we only need to hide the `Configure` button.